src/components/pages/schedule-page/schedule-event-edit-section/schedule-form/components/schedule-dropdown/TimeScheduleDropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useEffect } from 'react';
+import dayjs from 'dayjs';
+import { useField } from 'formik';
+
+import Dropdown from '@/components/common/ui/form/dropdown';
+import { FormikDropdownProps } from '@/components/common/ui/form/with-formik/dropdown/FormikDropdown';
+
+import { getDateWithTimeSetted } from '../../utils/getDateWithTimeSetted';
+
+import * as styles from './ScheduleDropdown.styles';
+
+interface TimeScheduleDropdownProps extends FormikDropdownProps {
+  date: dayjs.Dayjs;
+}
+
+export const TimeScheduleDropdown: FC<TimeScheduleDropdownProps> = ({
```

Comment:
we don't export components in such way. Should be:
`const TimeScheduleDropdown...`
`export default TimeScheduleDropdown`